packages/governance/test/unitTests/test-param-manager.js

```diff
@@ -17,48 +17,23 @@ test('params one Nat', async t => {
   const numberDescription = {
     name: numberKey,
     value: 13n,
-    type: ParamType.NAT,
   };
   const { getParam, updateNumber } = buildParamManager([numberDescription]);
   t.deepEqual(getParam(numberKey), numberDescription);
   updateNumber(42n);
   t.deepEqual(getParam(numberKey).value, 42n);
-
-  t.throws(
-    () => updateNumber(18.1),
-    {
-      message: '18.1 must be a bigint',
-    },
-    'value should be a nat',
-  );
-  t.throws(
-    () => updateNumber(13),
-    {
-      message: '13 must be a bigint',
-    },
-    'must be bigint',
-  );
 });
```

Comment:
These tests show the point of the enforcement. I think it's valuable to the voters and contract maintainer that they know that these calls wouldn't succeed.